src/Arachne.cc

```diff
@@ -1146,15 +1157,16 @@ testInit() {
     core.loadedContext = *core.localThreadContexts;
     core.loadedContext->wakeupTimeInCycles = ThreadContext::BLOCKED;
     *core.localOccupiedAndCount = {1, 1};
+    PerfStats::threadStats = std::unique_ptr<PerfStats>(new PerfStats());
 }
 
 /**
- * This function should be invoked in unit test teardown to clean up the state
- * that makes Arachne functions callable from the unit test.
+ * Tear down the state set up by mainThreadDestroy.
```

Comment:
Typo? "the state set up by mainThreadInit" not "Destroy". Also, probably mention that we destroy threadStats here.

Reply:
Yes on the typo. I think the new description of `mainThreadInit` should cover enough that we may not need to mention the destruction of threadStats.